### DISCUSSION TITLE
How to change RPU frame-rate

### DISCUSSION THREAD

**Discussion by ohhWTF** [quietvoid/dovi_tool#267] [Q&A] [answered]
Hi! Can anyone help me understand how to be able to change the frame-rate target of an extracted RPU? Example: I extract the RPU.bin from an HEVC @ 24fps and I then need to do the injection to an HDR10 HEVC @23.976fps. This means I need to change the RPU.bin fo 23.976fps to be able to match the target frame-rate of the HDR10. (this is the case with the recent release of "The Creator" - digital download with DV is 24fps, BD 4K release is only HDR10 and 23.976fps).  Thank you for the help! G.

**Comment by saindriches** [comment, marked as answer]
No changes are required if the only distinction between sources is the frame rate, with no difference in the content of each frame and the frame counts.

**Comment by ohhWTF** [comment]
Thank you for you answer, but I don't understand. Doesn't the total frame counts change if a same content is 23,976fps or 24fps? Thank you

**Reply by saindriches** [reply]
When a 24fps version exists, it usually indicates that the content was originally produced for cinemas, where these historical issues from NTSC do not exist. The common practice when a 23.976fps version is needed is to simply reduce the frame rate without making any adjustments to the frame sequence itself, while stretching the audio to slow it down, resulting in an elongation of 3.6 seconds for every hour of the original content.

**Reply by saindriches** [reply]
Anyway, check your specific content for possible rare cases that could prevent you from injecting the RPU.

**Comment by ohhWTF** [comment]
Ok, if you say that the frame sequence doesn't change, I'll try and find out if it works! Yes, I know how to deal with audio, no problem with that! Thank you again for you help and time. I'll let you know the results

**Reply by ohhWTF** [reply]
You were right! Injection worked perfectly! Thank you again! G.
